### PR TITLE
Add plan badges to enterprise features

### LINKED ISSUE #2287: Label paid features as such during the beta
Label certain features as Enterprise / Paid during Beta just to communicate to users what they're getting themselves into.

**Err on the side of _not_ labeling if we're unsure**

- [ ] Directory sync
- [ ] **(TBD)** Number of Gateways / Sites

### PR DESCRIPTION
**Pull request by jamilbk**
Fixes #2287 

Screenshot 2023-11-30 at 2 40 28 PM
Screenshot 2023-11-30 at 2 40 16 PM

### CODE REVIEW

**Comment by AndrewDryga** [comment]
I think we should also add them to Flows and Flow Activities?

BTW WDYT about renaming `enterprise` to `pro` or `business`? I think `enterprise` might look scary because some people will read it as "a looot of money" if all we have is `free` and `enterprise`.

File: elixir/apps/web/lib/web/live/settings/identity_providers/new.ex [hunk 1]
```diff
@@ -84,7 +84,7 @@ defmodule Web.Settings.IdentityProviders.New do
       adapter={@adapter}
       account={@account}
       name="OpenID Connect"
-      description="Authenticate users with a generic OpenID Connect adapter and synchronize users and groups with just-in-time provisioning."
+      description="Authenticate users with a universal OpenID Connect adapter and synchronize users just-in-time provisioning."
```
Comment (bmanifold) [inline elixir/apps/web/lib/web/live/settings/identity_providers/new.ex:87]: What do you think about removing `provisioning` at the end of this sentence?

**Review by bmanifold — APPROVED** [review]
Overall, LGTM.  Just one small question that can be ignored if you want.

File: elixir/apps/web/lib/web/live/settings/identity_providers/new.ex (hunk 1 above)
Comment (jamilbk) [reply elixir/apps/web/lib/web/live/settings/identity_providers/new.ex:87]: 👍

**Pull request merged**